packages/terra-hookshot/src/HookshotContent.jsx

```diff
@@ -216,3 +216,4 @@ HookshotContent.propTypes = propTypes;
 const onClickOutsideContent = onClickOutside(HookshotContent);
 
 export default onClickOutsideContent;
+export { HookshotContent };
```

Comment:
Export version of HookshotContent without onClickOutside HOC. Using popup overlay to handle onClickOutisde but keeping this HOC for passivity.